.github/actions/e2e_recover/action.yml

```
@@ -40,7 +40,7 @@ runs:
         start_time=$(date +%s)
         recovered=0
         while true; do
-            output=$(constellation recover --master-secret=${{ inputs.masterSecret }} --force)
+            output=$(constellation recover -C ${{ inputs.workspace }} --force)
```

```suggestion
            output=$(constellation recover --force)
```
And this doesn't have to se the workspace